src/Razor/src/Microsoft.VisualStudio.RazorExtension/SyntaxVisualizer/IntraTextAdornmentTagger.cs

```diff
@@ -143,8 +143,10 @@ private void HandleLayoutChanged(object sender, TextViewLayoutChangedEventArgs e
         // Filter out the adornments that are no longer visible.
         var toRemove = from keyValuePair
             in _adornmentCache
+#pragma warning disable format
             where !keyValuePair.Key.TranslateTo(visibleSpan.Snapshot, SpanTrackingMode.EdgeExclusive).IntersectsWith(visibleSpan)
             select keyValuePair.Key;
```

Comment:
What does this end up looking like if we format it? I can't see anything obviously weird here

Reply:
C# formatting bug?

![image](https://github.com/user-attachments/assets/be854c10-275d-4b70-92bd-ac6266b56110)

Reply:
... unless I manually indent ```in```, then formatter is happy with this

![image](https://github.com/user-attachments/assets/c212a3f9-cbde-4b18-b415-97a8f006e3b1)

Reply:
Looking at the docs at https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/from-clause, recommendation seems to be to have ```in``` on the same line as ```from```, after which formatting basically makes sense (though formatting in the first two examples in the same docs is still getting "corrected" by C# formatter). I made the change to make the formatter happy.

Reply:
ahh, yep, that's what it is. I didn't see it, but I always have put `in` on the same line as `from`.

Reply:
> ahh, yep, that's what it is. I didn't see it, but I always have put `in` on the same line as `from`.

FWIW, that's pretty uncommon.

Reply:
> FWIW, that's pretty uncommon

I think you misread what I wrote :) We're in vehement agreement about `from` and `in` being on the same line :)

>  I've personally never seen in on a separate line until now

Me neither, that's why I missed it when I wrote "I can't see anything obviously weird here" above.